Gems/Atom/RHI/DX12/Code/Include/Atom/RHI.Reflect/DX12/PlatformLimitsDescriptor.h

```diff
@@ -64,6 +64,12 @@ namespace AZ
             //! int array: Max count for descriptors 
             AZStd::unordered_map<AZStd::string, AZStd::array<uint32_t, NumHeapFlags>> m_descriptorHeapLimits;
 
+            // Number of max static handles for shader visible srv/uav/cbv views
+            uint32_t m_numShaderVisibleCbvSrvUavStaticHandles = 2000;
```

Comment:
How did you derive this value?

Reply:
This is a default value derived similar to other default values for various descriptor heaps (Check PlatformLimits.cpp within dx12 backend). It is data driven and can be overridden in the settings registry similar to other values. 2000 made the most sense based on the fact that we only use static handles for view that need to be cleared. I may increase the default value later on down the road if needed.